infra/build_specified_commit_test.py

```diff
@@ -41,9 +41,8 @@ def test_build_fuzzers_from_commit(self):
     The old commit should show the error when its fuzzers run and the new one
     should not.
     """
-
     with tempfile.TemporaryDirectory() as tmp_dir:
-      test_case = test_repos.TEST_REPOS[0]
+      test_case = test_repos.TEST_REPOS[1]
```

Comment:
Was this always wrong ?

Reply:
No, something changed in the outside world and these tests stopped working, because they are not written well.